PDFUnblocker.py

```
@@ -0,0 +1,63 @@
+from tkinter import Tk
+from tkinter.filedialog import askdirectory
+from tqdm import tqdm
+
+# DEFINE FUNCTIONS HERE
+
+def SingleUnlock(filename):
+    with pikepdf.open(filename,
+                      allow_overwriting_input=True) as pdf:
+        pdf.save(filename)
+
+def BatchUnlock(PDFList, NumberOfFiles):
```

Ruff

Missing return type annotation for public function `BatchUnlock`

Add return type annotation: `None`

(ANN201)